content/02.body.md

```diff
@@ -295,10 +295,10 @@ We expected features based on degree to show strong performance for a network re
   All five features show a positive relationship with degree, though the magnitude of this correlation is highly variable.
   The preferential attachment index is understandably perfectly correlated because it is equal to the product of source and target degree.
   Each panel indicates the Pearson correlation ("r") between feature and degree in the lower right corner.
-](https://github.com/greenelab/xswap-analysis/raw/f8dce1983243fd4056108c7c8bdcba895f6dfbaf/img/feature-degree.png){#fig:feature-degree width="100%"}
+](https://github.com/greenelab/xswap-analysis/raw/4f06bdaf1f034af9136e25c03f9891a145b9bf91/img/feature-degree.png){#fig:feature-degree width="100%"}
 
 <!--
-fig:feature-degree is created by 
+fig:feature-degree is created by
 https://github.com/greenelab/xswap-analysis/blob/b0db22c82b2e58bf1ef5ae78317167982016e26b/nb/7.fig5.feature_degree/2.plot_correlation.ipynb
```

Comment:
I think we should update these commit hashes as well. Could just do a find and replace all for `b0db22c82b2e58bf1ef5ae78317167982016e26b`